helm/amster/templates/amster.yaml

```diff
@@ -20,7 +20,20 @@ spec:
         app: {{ template "fullname" . }}
         component:  {{ .Values.component }}
     spec:
-      terminationGracePeriodSeconds: 5    
+      terminationGracePeriodSeconds: 5   
+      initContainers:
+      - name: git-init
+        image: forgerock/git:6.0.0
```

Comment:
should this be 6.5.0?

Reply:
Eventually. Right now there have not been any changes